### ISSUE TITLE
Feat: Merge Queue, Status, Type, and other `TOP LEVEL` Labels

### ISSUE THREAD

**Issue by sambacha** [sambacha/polemic-labels#5]
> [!IMPORTANT]
> Consider category 'queue' to encompass commit, build, and other incremental build steps    

## Status Labels

- `status: needs discussion` - needs further discussion by contributors to triage
- `status: needs clarification` - needs clarification from the author to triage
- `status: needs investigation` - needs investigation by contributors to triage
- `status: needs reproducible example` - needs a reproducible repo or demo by the author to triage
- `status: wip` - is being worked on by someone
- `status: blocked` - is blocked by another issue or pull request

## Merge Queue Labels
~## Pull Request Labels~

~- `pr: needs revision` - pull request needs revision by the author~
~- `pr: blocked` - pull request is blocked by another issue or pull request~
~- `pr: needs documentation` - pull request needs documentation adding by the author~
~- `pr: needs tests` - pull request needs tests adding by the author~
~- `pr: dependencies` - pull request relates to dependencies~

## Syntax Labels

- `syntax: $lang` - relates to a language

> Example
> - `syntax: scss` - relates to SCSS and SCSS-like syntax

## Priority Labels

- `priority: high` - is impactful on users

## Type Labels

- `type: documentation` - an improvement to the documentation
- `type: enhancement` - a new feature that isn't a rule, option or autofix
- `type: new rule` - a new rule
- `type: new option` - a new option for an existing rule
- `type: new autofix` - autofix support for an existing rule
- `type: bug` - a problem with a feature, rule, option or autofix
- `type: infrastructure` - an improvement to the developer operations like the CI
~- `type: meta` - an improvement to collaboration platform like GitHub issues ~
- `type: tests` - an improvement to testing

## Contribution Labels

- `good first issue` - is good for newcomers
- `help wanted` - a helping hand is wanted
